operator/v1/types_ingress.go

```diff
@@ -716,8 +720,58 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
+	// eipAllocations cannot be used with "Internal" scope.
+	// An EIP can be allocated to only a single IngressController.
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
+	// information about configuration, characteristics, and limitations of Elastic IP addresses.
+	// +openshift:enable:FeatureGate=SetEIPForNLBIngressController
+	// +optional
+	// +listType=atomic
+	// Each entry in the eipAllocations list must be unique
+	// +kubebuilder:validation:XValidation:rule=`self.all(x, self.exists_one(y, x == y))`,message="eipAllocations cannot contain duplicates"
+	// A maximum of 10 IP allocations are permitted
+	// +kubebuilder:validation:MaxItems=10
```

Comment:
The text in the generated yaml is not well formatted.  Please order your sentences logically (group the list of limitations together, and then reference further documentation), and add line breaks.  
```suggestion
	// eipAllocations assigns Elastic IP addresses to the Network Load Balancer.
	// 
	// The following restrictions apply:
	// 
	// - eipAllocations cannot be used with "Internal" scope.
	// 
	// - The number of EIPs must match the number of subnets that are used for the load balancer.
	// 
	// - An EIP can be allocated to only a single IngressController.
	// 
	// - Each entry in the eipAllocations list must be unique.
	// 
	// - A maximum of 10 EIPs are permitted.
	// 
	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
	// information about configuration, characteristics, and limitations of Elastic IP addresses.
	// +openshift:enable:FeatureGate=SetEIPForNLBIngressController
	// +optional
	// +listType=atomic
	// +kubebuilder:validation:XValidation:rule=`self.all(x, self.exists_one(y, x == y))`,message="eipAllocations cannot contain duplicates"
	// +kubebuilder:validation:MaxItems=10
```

Reply:
ok done

Reply:
just have removed hyphens as per  @JoelSpeed  's suggestion

Reply:
just added hyphens back https://github.com/openshift/api/pull/1826#discussion_r1678346598 as per @gcs278 comment.

Reply:
Sorry, to be clear, my comment was _not_ suggesting to use bullets, but to write it out in sentences as @JoelSpeed suggested. I was trying to understand why there were still newlines between every comment, which has no effect.

> From my experience, you need to put - or * in front of each item, BUT that still doesn't produce a new line after the last bullet, \*\*so I don't think we should use bullets.\*\*
>
> Personally, I'd just write this out as a paragraph to avoid any confusion here. It may confuse people into thinking the line breaks worked, or used to work.

The last bullet does not provide a newline in `oc explain`, which I still consider problematic with the previous [fix](https://github.com/openshift/api/commit/29627755e664fe70ba9be7a7d78302b920560291#diff-2101eac4196d9b14cf061c8a6a4d40f9d8e5a77fc2690f969e7293294218afe3L59).